app/html/src/server/framework-preset-html.ts

```diff
@@ -20,3 +21,10 @@ export function webpack(config: Configuration) {
     },
   };
 }
+
+export function babelDefault(config: TransformOptions) {
+  return {
+    ...config,
+    plugins: [...config.plugins, require.resolve('@babel/plugin-transform-react-jsx')],
+  };
+}
```

Comment:
This needs to be applied to the babel config used to jest as well.

I've tried a few methods, but didn't get it to work..

Reply:
I'll take a look upcoming weekend. Thanks for suggestion 🚀